index.css

```diff
@@ -0,0 +1,216 @@
+}
+
+li {
+    display: inline;
+    margin-right: 20px;
+}
+
+/* Style the links */
+a {
+    text-decoration: none;
+    color: black
+}
+
+img {
+    height: 24px;
+}
+
+.title {
+    height: 25%;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: end;
+
+}
+
+.title img {
+    height: 50%;
+
+}
+
+.search {
+    height: 25%;
+    /* background-color: red; */
+}
+
+.language {
+    height: 30%;
+    /* background-color: blue; */
+}
+
+.footer {
+    height: 12%;
+    background-color: #f2f2f2;
+    display: flex;
+    flex-direction: column;
+}
+
+.footer_country {
+    height: 50%;
+    border-bottom: 1px solid #dadce0;
+}
+
+.footer_terms {
+    height: 50%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.footer_text {
+    color: #70757a;
+    padding: 10px 30px;
+}
+
+.footer_text li a {
+    color: #70757a;
+}
+
+.language_choices {
+    height: 20%;
+    width: fit-content;
+    margin: 0px auto;
+    /* background-color: orange; */
+    display: flex;
+    flex-direction: row;
+}
+
+.language_choices li {
+    margin: 0px 5px;
+}
+
+.language_choices li a {
+    color: #1a0dab;
+    font-size: small;
+}
+
+.language_choices_text {
+    color: #4d5156;
+    white-space: nowrap;
+}
+
+.searchbar {
+    /* background-color: orange; */
+    height: 60%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.buttons {
+    /* background-color: orangered; */
+    height: 40%;
+    width: fit-content;
+    margin: 0px auto;
+}
+
+.buttons_search {
+    height: 40px;
+    width: fit-content;
+    background-color: #f8f9fa;
+    color: #3a3c42;
+    border: none;
+    padding: 10px 20px;
+    font-size: 14px;
+    margin: 0px 5px;
+    border-radius: 4px;
+    border: 1px solid transparent;
+    box-shadow: 0 1px 1px rgba(0, 0, 0, 0);
+}
+
+.buttons_search:hover {
+    box-shadow: 0 1px 1px rgba(0, 0, 0, .1);
+    border: 1px solid #dadce0;
+    cursor: pointer;
+    color: #202124
+}
+
+.text {
+    height: auto;
+    /* Set a fixed height for the textarea */
+    width: 36.7em;
+    border: none;
+    border-radius: 30px;
+    font-size: 16px;
+    outline: none;
+    margin: 0px 40px;
+    white-space: normal;
+    overflow-y: auto;
+    display: flex;
+    align-items: center;
+    /* Center vertically */
+    padding: 10px;
+    /* Add some padding to improve appearance */
+    resize: none;
+}
+
+
+
+
+.search_img {
+    position: absolute;
+    left: 7px;
+    bottom: 10px;
+    height: 20px;
+}
+
+.google_lens {
+    position: absolute;
+    right: 5px;
+    bottom: 10px;
+}
+
+.content {
+    height: 40%;
+    width: 41em;
+    border: 1px solid #ccc;
+    border-radius: 30px;
+    font-size: 16px;
+    outline: none;
+    position: relative;
+}
+
+.content:hover {
+    box-shadow: 1px 1px 10px rgba(0, 0, 0, .1);
+    border: 1px solid #dadce0;
+    cursor: pointer;
+    color: #202124
+}
```

Comment:
Please ensure you have an extra line at then end of file (EOF), this ensure the other person who need to update the code doesn't update the history of the code that you have written.

Reply:
https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log